js/src/frameworks/langchain.ts

```diff
@@ -48,7 +49,7 @@ export class LangchainToolSet extends BaseComposioToolSet {
 
         const parameters = jsonSchemaToModel(schema["parameters"]);
 
-        // @TODO: Add escriiption an othjer stuff here
+        // @TODO: Add escriiption an other stuff here
```

Comment:
There's a typo in the TODO comment. 'escriiption an othjer' should be 'description and other'.